examples/Cargo.toml

```diff
@@ -11,6 +11,7 @@ members = [
     "json/core",
     "password-checker",
     "password-checker/core",
+    "prorata/host",
```

Comment:
This looks different from the other examples but let the example use the same overall layout as our template. Is that a reasonable practice?